### DISCUSSION TITLE
Can a external cluster (EKS) be added in two argocd instance running in two different EKS

### DISCUSSION THREAD

**Discussion by avinavgit** [argoproj/argo-cd#10894] [General]
Hello,

We have a scenario where we need to add a cluster in two different argocd instance running on different clusters, Is it possible?

**Comment by jgwest** [comment]
I've not tried itself, but I don't know see a reason why it wouldn't work. You just need to make sure that the Argo CD instances don't interfere with each other, for example, by deploying the same resources to the same Namespaces, or overriding shared cluster-scoped resources (CRDs, CR/Bs, etc).
